release/internal/operator/operator.go

```diff
@@ -14,6 +14,18 @@ const (
 	Registry  = registry.QuayRegistry
 )
 
+// ImageInfo returns the registry and image name for the operator images.
+// When a registry is specified in the config, the image name is modified and the registry is used
+func ImageInfo(registryRpl string) (string, string) {
+	registry := Registry
+	imageName := ImageName
+	if registryRpl != "" {
+		registry = registryRpl
+		imageName = strings.ReplaceAll(imageName, "/", "-")
```

Comment:
I still don't think this is appropriate - why do we need to do this? We should just use the image unmodified. 

```go
const (
  ImageName = "operator"
  Registry = "quay.io/tigera"
)
```

Reply:
Operator images are built and pushed for each hashrelease becasue calico version needs to be specifed in the operator.
While testing in my fork, I need to push operator to something other than `tigera/operator` so i current have this to standardize it to `<repo>/tigera-operator` as the image name can't still be `tigera/operator`

Reply:
This ReplaceAll command is just too much magic for my liking and makes it harder to understand what is happening and why. If when testing in your fork you need a different image name, then we should just make that a configuration option so that you can specify what you need.

Reply:
I have created an OperatorConfig to allow configuring operator variables i.e. image, registy, git repo & branch